### DISCUSSION TITLE
Connecting the cable on 52Pi N04

### DISCUSSION THREAD

**Discussion by WhoIsAvailable** [geerlingguy/raspberry-pi-pcie-devices#656] [General]
Last week, I bought a 52Pi N04 top HAT which I want to install onto my Pi 5 and active cooler. However, the instructions that came with the HAT are incomplete and out of date. I tried to get support from the 52Pi website through their support email address and chat line with no response.

My main problem is the FPC / FFC cable(s) that is intended to connect the HAT to the PCIe slot on the Pi 5. The instructions tell me that the contacts on the cable should face one way, which is easy enough to achieve, even though the contacts are now silver where the instructions say they are gold. The problem is that there is no guidance on which end of the cable should be inserted into which slot – especially given that there are now distinguishing marks on the two ends of the cable (a digit 1 on one end and digits 6 and 8 on the other).

I did find a video on youtube showing the installation of a 52Pi N04 HAT, but that clearly shows that my cables are different from the ones in the video.

Perhaps it does not matter which way round the cable is inserted, but I have no inclination to experiment and possibly damage the Pi given the ease with which electronic components can be damaged.

Can anyone here please either give me guidance on which way round the cable should be inserted, or confirm that it really does not matter ?

Thanks in advance for any help . . .

**Comment by yoyojacky** [comment]
> Last week, I bought a 52Pi N04 top HAT which I want to install onto my Pi 5 and active cooler. However, the instructions that came with the HAT are incomplete and out of date. I tried to get support from the 52Pi website through their support email address and chat line with no response.
> 
> My main problem is the FPC / FFC cable(s) that is intended to connect the HAT to the PCIe slot on the Pi 5. The instructions tell me that the contacts on the cable should face one way, which is easy enough to achieve, even though the contacts are now silver where the instructions say they are gold. The problem is that there is no guidance on which end of the cable should be inserted into which slot – especially given that there are now distinguishing marks on the two ends of the cable (a digit 1 on one end and digits 6 and 8 on the other).
> 
> I did find a video on youtube showing the installation of a 52Pi N04 HAT, but that clearly shows that my cables are different from the ones in the video.
> 
> Perhaps it does not matter which way round the cable is inserted, but I have no inclination to experiment and possibly damage the Pi given the ease with which electronic components can be damaged.
> 
> Can anyone here please either give me guidance on which way round the cable should be inserted, or confirm that it really does not matter ?
> 
> Thanks in advance for any help . . .

please send email to sales@52pi.com and cc to jacky.li@52pi.com , and paste your order number please. we are going to help you out.

**Comment by the-polak** [comment]
Ribbon cable? Gold side down. Gently flip the tabs. Helps to attach the ribbon cable prior to attaching standoffs. See pics for reference. 

![IMG_6315](https://github.com/user-attachments/assets/f001768e-dd39-4b44-af86-6b7ca8a24999)

![IMG_6314](https://github.com/user-attachments/assets/b26618ef-32b9-4daf-973d-da6c83f59909)

![IMG_6313](https://github.com/user-attachments/assets/51c7e640-6673-4b66-b553-e2c5ce32fc0d)

**Reply by yoyojacky** [reply]
yes, absolutly right! nice picture !!!
